src/utils/getUserById.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from '../types/User';
+import usersFromServer from './../api/users';
+
+export const getUserById = (userId: number): User | undefined => {
```

Comment:
Suggestion: I think `User | undefined` will be better use `User | null` and in return you can add `|| null`

Reply:
When I write null, I get a bunch of errors, which saying "Type 'null' is not assignable to type 'User | undefined'", because in todos there are no users. So I must write undefined. But if there are users set to null, of course I type the return to User | null